plugins/graphiql/src/lib/api/GraphQLEndpoints.ts

```diff
@@ -76,6 +77,30 @@ export class GraphQLEndpoints implements GraphQLBrowseApi {
     };
   }
 
+  // Create the GraphQLEndpoints from the catalog entities with kind=api and type = graphql.
+  static async fromCatalogEntities(
```

Comment:
This should not be a public method. The API already has a `getEndpoints` which is the one that should return the results. The fact that they came from one source or another should be entirely hidden to the `GraphiQLPage`. Make it the responsibility of the class to initialize itself in any way it sees fit and hide how it deals with the enumeration of urls